playbooks/provisioner/openstack/main.yaml

```diff
@@ -0,0 +1,6 @@
+---
+- name: Provision OpenStack resources
+  hosts: localhost
+  roles:
+      - openstack/provision/network
```

Comment:
I would much more prefer to stick to OpenStack naming (neutron, nova, glance, cinder)...